aiida_quantumespresso/parsers/cp.py

```diff
@@ -39,140 +40,181 @@ def parse(self, **kwargs):
         elif len(xml_files) > 1:
             return self.exit(self.exit_codes.ERROR_OUTPUT_XML_MULTIPLE)
 
-        if self.node.process_class._FILE_XML_PRINT_COUNTER_BASENAME not in list_of_files:
-            self.logger.error('We could not find the print counter file in the output')
-            # TODO: Add an error for this counter
-            return self.exit(self.exit_codes.ERROR_MISSING_XML_FILE)
+
+        print_counter_xml=True
+        no_trajectory_output=False
+        if self.node.process_class._FILE_XML_PRINT_COUNTER_BASENAME not in list_of_files and self.node.process_class._FILE_PRINT_COUNTER_BASENAME not in list_of_files:
+            self.logger.error('We could not find the print counter file in the output (' + self.node.process_class._FILE_PRINT_COUNTER_BASENAME + ' or ' + self.node.process_class._FILE_XML_PRINT_COUNTER_BASENAME + ' not in {} )'.format(list_of_files) + 'assuming no trajectory output was produced' )
+            no_trajectory_output=True
+            #this can happen and it is not an error!
+        if not no_trajectory_output:
```

Comment:
Maybe at least reference one version in which you know the given format was used